Demo/UITests/BraintreeDropIn_EditMode_UITests.swift

```diff
@@ -23,15 +23,15 @@ class BraintreeDropIn_EditMode_UITests: XCTestCase {
     }
 
     func testDropIn_canDisplay_editScreen() {
-        self.waitForElementToBeHittable(app.buttons["Edit"])
-        app.buttons["Edit"].forceTapElement()
+        self.waitForElementToAppear(app.buttons["Edit"])
+        app.buttons["Edit"].firstMatch.forceTapElement()
```

Comment:
These tests were finding two "Edit" buttons for some reason, even though when I look in the view debugger I only see one.